dist/radio/ds6/radio.css

```diff
@@ -0,0 +1,110 @@
+.radio {
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  font-size: 1rem;
+  vertical-align: text-bottom;
+}
+.radio__control[type="radio"],
+.radio__custom-control[type="radio"] {
+  font-size: 100%;
+  height: 1.125em;
+  opacity: 0;
+  position: absolute;
+  width: 1.125em;
+  z-index: 1;
+}
+.radio__custom-control[type="radio"]:not([disabled]) {
```

Comment:
We could maybe make better use of grouped selectors for the next 20 lines or so, but I don't thin it's a blocker. @seangates thoughts?